src/planscape/planning/serializers.py

```diff
@@ -598,7 +598,24 @@ def validate(self, data):
         return data
 
     def validate_geometry(self, value):
-        geojson_serializer = GeoJSONSerializer(data=json.loads(value))
+        # consolidate a list of feature collections into one object
+        ## TODO: use built-in approaches for this
+        if isinstance(value, list):
```

Comment:
One of the upload issues we saw is that shpjs parses some uploads into an array of FeatureCollection objects.
Something like this code merges these into a single object, but maybe there are preferred ways of dealing with this? Or maybe this conversion isn't actually necessary?